src/Components/People.js

```diff
@@ -0,0 +1,90 @@
+import React, { Component } from "react";
+import axios from "axios";
+
+ class People extends Component {
+  constructor() {
+    super();
+    this.state = {
+      searchPeople: "",
+      currentPeople: {},
+      isError: false,
+      savedName: "",
+    };
+  }
+
+  handleChange = (e) => {
+    this.setState({
+      searchPeople: e.target.value,
+    });
+  };
+
+  handleSubmit = async (e) => {
+    e.preventDefault();
+    const { searchPeople } = this.state;
+    try {
+      const { data } = await axios.get(
+        "https://ghibliapi.herokuapp.com/people"
+      );
+    
+      const person = data.find((element) => element.name === searchPeople);
```

Comment:
I like the strategy you use: when a user submits a form, fetch all of the people from the ghibli api, and find the actual person we want.

Could you come up with an alternate strategy where you only make one api call in componentDidMount() ? Then when a user submits a search, you only have to search the data you've already loaded from the ghibli api instead of waiting for an api call to complete first.